icu4j/main/classes/core/src/com/ibm/icu/impl/number/parse/IgnorablesMatcher.java

```diff
@@ -18,6 +18,9 @@
     public static final IgnorablesMatcher STRICT = new IgnorablesMatcher(
             StaticUnicodeSets.get(StaticUnicodeSets.Key.STRICT_IGNORABLES));
 
+    public static final IgnorablesMatcher COMPATIBILITY = new IgnorablesMatcher(
```

Comment:
Suggestion: `JAVA_COMPATIBILITY`

Then the use case is specific *and* we don't have to port to C++.

Reply:
Done. PTAL